test/unit/query/sort.js

```diff
@@ -50,12 +51,30 @@ var expected = [
 
 module.exports.tests.query = function(test, common) {
   test('valid part of query', function(t) {
-    var sort = generate;
+    var sort = generate();
     t.deepEqual(sort, expected, 'valid sort part of the query');
     t.end();
   });
 };
 
+module.exports.tests.queryWithInput = function ( test, common ){
```

Comment:
* test for when params object has no ```input``` or when ```input = ''```
* test for when params object is not passed in